code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -199,6 +199,9 @@
 			FindTarget()
 
 /mob/living/simple_animal/hostile/proc/AttackingTarget()
+	if(target == src)
+		to_chat(src,"Why attack yourself?")

```

Comment:
Mhm, perhaps: "You were about to attack yourself, but chose not to." The gist of the message is similar, but it sounds much better. Or perhaps in present tense if that sounds better.